.github/CODEOWNERS

```diff
@@ -45,6 +45,7 @@ extension/observer/                        @open-telemetry/collector-contrib-app
 internal/awsxray/                          @open-telemetry/collector-contrib-approvers @anuraaga @mxiamxia
 internal/k8sconfig/                        @open-telemetry/collector-contrib-approvers @pmcollins @asuresh4
 internal/splunk/                           @open-telemetry/collector-contrib-approvers @pmcollins @asuresh4
+internal/stanza/                           @open-telemetry/collector-contrib-approvers @djaglowski
```

Comment:
Nit: should we go with `internal/logreceiverhelper` as the issue suggests? I am worried that "stanza" is not self-explanatory for future contributors.

Reply:
I debated the same. One reason I had for not naming it as such is that non-stanza based log receivers wouldn't be able to make use of this, and I didn't want to imply that this was the only way to make a log receiver. That said, this is something we can deal with later on as necessary.

Reply:
SGTM. Let's keep for now.